### ISSUE TITLE
Need a clarification regarding train/validation split.

### ISSUE THREAD

**Issue by ashesh-0** [jiupinjia/Deep-adversarial-decomposition#14]
Hi,
I computed the PSNR using your pre-trained checkpoint file on dogsflowers dataset. I'm able to get the same PSNR on the validation set. However, I find that PSNR on validation data is better than PSNR on training data which is surprising to me. I evaluated the model 3 times each on training and validation data. I get `31.41+-0.01` on validation and `30.46+-0.01` on training data. 

Secondly, when I train the model from scratch, my validation PSNR reaches only `24` where as the training PSNR reaches `31`. 
I'm sharing the notebook which I've used to compute the PSNR. I'm also sharing the screeshots of the notebook when evaluated on training data and on validation data. PSNR is gettiing reported on the output of 7th cell. 

For flowers data, I've used 'trn1' and 'val1' for training and validation splits respectively. For the dogs, I randomly separated the images into two groups  while keeping 20% of data for validation and 80% for training. 

It would help me immensely if you could explain why the training data performs poorly vis a vis the validation data or where I'm doing things incorrectly :). 
[Notebook.zip](https://github.com/jiupinjia/Deep-adversarial-decomposition/files/8514174/Notebook.zip)

# On Validation data
![DeepAdvDec_dogsflowers_pretrained](https://user-images.githubusercontent.com/6098025/164035345-397ff754-306d-43df-a23a-673f6bf6b80e.png)

# On Training Data
![DeepAdvDec_dogsflowers_pretrained_traindata](https://user-images.githubusercontent.com/6098025/164035403-9879731b-12ac-49c5-925c-8942c71ad902.png)
